projects/storefrontlib/src/layout/a11y/keyboard-focus/index.ts

```diff
@@ -4,12 +4,14 @@ export * from './focus.directive';
 export { FocusConfig, TrapFocus, TrapFocusType } from './keyboard-focus.model';
 export * from './keyboard-focus.module';
 export * from './services/index';
+export * from './config/index';
 
 // export * from './autofocus/index';
 // export * from './base/index';
 // export * from './block/index';
 // export * from './escape/index';
 // export * from './lock/index';
+// export * from './on-navigate/index';
```

Comment:
Shouldn't we to be more specific with this name? Maybe we can call it more like `focus-reset`?

Reply:
`on-navigate` contains focus functions related to the navigation itself. `focus-reset` could be considered just one of these features as seen in the config and we could add more navigation-related features later.